WebSharper.UI.Next/Doc.fs

```diff
@@ -599,23 +599,30 @@ type [<JavaScript; Proxy(typeof<Doc>); CompiledName "Doc">]
                 (Attr.Concat attr |> Attr.Append atType, "input")
             | TextArea -> (Attr.Concat attr, "textarea")
         let el = DU.CreateElement elemTy
-        let valAttr = Attr.Value var
-        Doc'.Elem el (Attr.Append attrN valAttr) Doc'.Empty
+        Doc'.Elem el (Attr.Append attrN value) Doc'.Empty
 
     static member Input attr (var: Var<string>) =
-        Doc'.InputInternal attr (var : Var<string>) SimpleInputBox
+        Doc'.InputInternal attr (Attr.Value var) SimpleInputBox
 
     static member PasswordBox attr (var: Var<string>) =
-        Doc'.InputInternal attr var (TypedInputBox "password")
+        Doc'.InputInternal attr (Attr.Value var) (TypedInputBox "password")
 
     static member IntInput attr (var: Var<int>) =
-        Doc'.InputInternal attr var (TypedInputBox "number")
+        let parseInt s =
+            let pd = JS.ParseInt(s)

```

Comment:
Use `JS.ParseInt(s, 10)` to avoid base issues. For example a string starting with a `0` will be parsed as octal instead of decimal, so `parseInt "010"` will return `Some 8` instead of `Some 10`. Same with `ParseFloat` below.